src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -25,6 +25,42 @@ class UserInfoSection extends React.Component {
       );
     }
 
+    let entitlementInfo;
+    if (enrollmentData.percentageBenefit === 0 && enrollmentData.originalEntitlement === 0) {
+      entitlementInfo = (
+        <div>
+          <h4>When You Can Receive Benefits</h4>
+          <div className="usa-alert usa-alert-warning usa-content">
+            <div className="usa-alert-body">
+              <h2>Currently Disallowed</h2>
+              We have received your application and have determined that you are not currently eligible
+              for Post-9/11 GI Bill benefits. Additional service time could change this determination.
+            </div>
+          </div>
+        </div>
+      );
+    } else {
+      entitlementInfo = (
+        <div>
+          <div>
+            <h4>When You Can Receive Benefits</h4>
```

Comment:
Ah I misunderstood your comment when I first read it. It might be tricky to do this because there are actually 3 types of data being rendered under entitlement:
1. When You Can Receive Benefits
2. Your Benefit Level
3. Months received, used, and remaining

I could create 3 separate variables for each of these, but the reason why I wanted to group them is to show that either all 3 of these pieces are rendered, or none. If none, instead the disallowed message appears. I think this way it's a bit more clear when these things should be rendered, even if we're grouping 3 disparate pieces of information together. What do you think?